.travis.yml

```diff
@@ -12,7 +12,7 @@ before_script:
 script:
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then git diff origin/master -U0 README.md | grep -Pos "(?<=^\+).*" >> temp.md; fi || (exit 0)'
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then node tests/test.js temp.md; else node tests/test.js README.md; fi'
-  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then if [ -f temp.md ]; then awesome_bot temp.md --allow-redirect --skip-save-results --allow 202; else (exit 0); fi else awesome_bot README.md --allow-redirect --skip-save-results --allow 202; fi'
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then if [ -f temp.md ]; then awesome_bot temp.md --allow-redirect --skip-save-results --allow 202 --white-list airsonic.github.io/docs/apps,github.com/bblfsh/bblfshd; else (exit 0); fi else awesome_bot README.md --allow-redirect --skip-save-results --allow 202 --white-list airsonic.github.io/docs/apps,github.com/bblfsh/bblfshd; fi'
```

Comment:
Please don't whitelist bblfshd, the duplicate Source Code link should be removed

Reply:
I've adjusted README.md and .travis.yml to fix this.